src/idlemonitor/DefaultIdleMonitor.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import PingPong from '../pingpong/PingPong';
+import PingPongObserver from '../pingpongobserver/PingPongObserver';
+import IdleMonitor from './IdleMonitor';
+
+export default class DefaultIdleMonitor implements IdleMonitor, PingPongObserver {
+  private canReceivePongs: boolean = false;
+  private lastPongResponseSeen: number = Date.now();
```

Comment:
- Rename it to `lastActiveTimestampMs` because, technically, you don't get a pong when instantiating this class.

  `__TimestampMS` postfix examples in SDK
  ```
  connectionStartTimestampMs
  currentTimestampMs
  lastGoodSignalTimestampMs
  lastNoSignalTimestampMs
  lastPacketLossInboundTimestampMs
  lastWarnTimestampMs
  lastWeakSignalTimestampMs
  previousTimestampMs
  setLastGoodSignalTimestampMs
  setLastNoSignalTimestampMs
  setLastPacketLossInboundTimestampMs
  setLastWeakSignalTimestampMs
  startTimestampMs
  videoInputAttachedTimestampMs
  ```